pkg/azureclients/privateendpointclient/interface.go

```diff
@@ -20,15 +20,26 @@ import (
 	"context"
 
 	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-02-01/network"
+
+	"sigs.k8s.io/cloud-provider-azure/pkg/retry"
+)
+
+const (
+	// APIVersion is the API version for network.
+	APIVersion = "2021-02-01"
+	// AzureStackCloudAPIVersion is the API version for Azure Stack
+	AzureStackCloudAPIVersion = "2019-03-01"
```

Comment:
I was using the same one as privateLinkService client. The version can be found here: https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-version-profiles-go?view=azs-2108

Reply:
o, that's great, could you put this link to the comments? thanks

Reply:
I'm checking with the original author, Jesus, from Azure Stack team. I'll update it after confirmation.

Reply:
Confirmed, above link is the source of truth but we find they do not support these resources in Azure Stack cloud so we leave an error instead of using any version. I already updated the comment.